vscode/src/language-service/codeLens.ts

```diff
@@ -7,7 +7,7 @@ import {
   qsharpLibraryUriScheme,
 } from "qsharp-lang";
 import * as vscode from "vscode";
-import { toVscodeRange } from "../common";
+import { toVsCodeRange } from "../common";
```

Comment:
Okay, but now the capitalization doesn't match the other `toVscode*` functions from that module. (Also I totally think of vscode as one word...)

Reply:
Ah, I was trying to match `toVsCodeDiagnostic`, and also vs code's branding which states that it is "VS" and then "Code" as two words. I can revert this if you feel strongly

Reply:
Dammit I didn't notice `toVsCodeDiagnostic` :( I just wanted them all to match and was trying to save you some unnecessary changes. I don't really mind which spelling we choose as long as we're converging.

Reply:
Would you rather me revert this, or perhaps do a quick rename-symbol on the rest?

Reply:
I don't mind if you rename-symbol the rest, since that would make them all match